Refrigerator/Program.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Remoting.Services;
+using System.Text;
+using System.Threading.Tasks;
+using static Refrigerator.Refrigerator;
+
+namespace Refrigerator
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            List<Refrigerator> refrigerators = new List<Refrigerator>();
+            Refrigerator refrigerator1 = new Refrigerator(1, "Model A", "White", 5);
+            Refrigerator refrigerator2 = new Refrigerator(2, "Model B", "Black", 5);
+            Refrigerator refrigerator3 = new Refrigerator(3, "Model c", "Green", 5);
+
+            refrigerators.Add(refrigerator1);
+            refrigerators.Add(refrigerator2);
+            refrigerators.Add(refrigerator3);
+
+
+            while (true)
```

Comment:
what is the stop condition here ? 
please make it clear in this part of the code. 
while (true) --> is a bad practice.
try to put boundaries of your code and understand when it should stop, or crash.